README.md

```diff
@@ -63,4 +63,9 @@ npm run lint:watch
 ### Provided npm commands
 * npm start -> It will start web app on localhost:3000 using debug api key and any change will reload server (thx to [nodemon](https://github.com/remy/nodemon))
 * npm test -> It will execute unit, integration tests and unit tests code coverage check.
-* npm run prod -> Remove dev dependencies and install production dependencies if needed
+* npm run test:watch -> It starting file watcher who will trigger tests if any file will change
```

Comment:
Not to be picky, but this one is a little bit unclear - maybe something like this would sound better? "It will start a file watcher that will trigger tests if any of the files are changed".